packages/editor/src/client/routes/articleEditor.tsx

```diff
@@ -192,6 +192,24 @@ export function ArticleEditor({id}: ArticleEditorProps) {
     }
   }
 
+  // Reads title and lead from the first block and saves them in variables
+  function syncBlocksWithMeta() {
```

Comment:
I think it's better if you are more explicit with the title. Maybe `syncFirstTitleBlockWithMetadata`